src/test/integration/translation-validation.test.ts

```
@@ -0,0 +1,152 @@
+    };
+  }
+
+  function getTranslationPairs(articles: typeof mockArticles) {
+    const pairs = new Map<string, string[]>();
+    
+    articles.forEach(post => {
+      const id = post.data.translationId;
+      if (!pairs.has(id)) {
+        pairs.set(id, []);
+      }
+      pairs.get(id)!.push(`${post.data.lang}:${post.slug}`);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)